migrations/2_deploy_contracts.js

```diff
@@ -21,14 +21,15 @@ module.exports = function(deployer, network) {
             return;
         }
 
-        await deployer.deploy([[ENSRegistry], [DummyDNSSEC]]);
+        await deployer.deploy(ENSRegistry);
```

Comment:
Any reason for this change?

Reply:
yes, because of latest truffle. Bundled deploys no longer exist